opentelemetry-instrumentation/src/opentelemetry/instrumentation/bootstrap.py

```diff
@@ -35,7 +35,6 @@
 instrumentations = {
     "aiohttp-client": "opentelemetry-instrumentation-aiohttp-client>=0.15b0",
     "aiopg": "opentelemetry-instrumentation-aiopg>=0.15b0",
-    "asgi": "opentelemetry-instrumentation-asgi>=0.11b0",
```

Comment:
why are these instrumentations being removed?

Reply:
We are moving them into opentelemetry.util.